include/ignition/gazebo/gui/GuiEvents.hh

```diff
@@ -235,6 +238,28 @@ namespace events
     /// \brief Private data pointer
     IGN_UTILS_IMPL_PTR(dataPtr)
   };
+
+  /// \brief Event that notifies a visual plugin is to be loaded
+  class IGNITION_GAZEBO_GUI_VISIBLE VisualSdfPlugin: public QEvent
```

Comment:
This could be plural and hold `sdf::Plugins` to address #1363

Reply:
[61fcf05](https://github.com/gazebosim/gz-sim/pull/1352/commits/61fcf059bdda7db75e59e12058eec67f713d5fed)